assets/style.styl

```diff
@@ -54,6 +54,10 @@ tour-tooltip
     height 0
     position absolute
 
+  // Hidden
+  &.inactive
+    display: none
+
```

Comment:
Unrelated fix since the hidden tooltip was causing UI issues since it was still layered above other elements in the UI.